yarn-project/noir-contracts.js/package.json

```diff
@@ -1,20 +1,20 @@
 {
-  "name": "@aztec/noir-contracts",
+  "name": "@aztec/noir-contracts.js",
   "version": "0.1.0",
   "type": "module",
   "exports": {
     ".": "./dest/src/index.js",
-    "./target/*": "./dest/target/*.json",
+    "./artifacts/*": "./dest/artifacts/*.json",
```

Comment:
Which script copies artifacts to this folder?

Reply:
NVM, found it https://github.com/AztecProtocol/aztec-packages/pull/4479/files#diff-e26b6aae25be3c811626d6c09123b098b49f836c476f88e0a0270bccfe24f56bR27